backend/battles/tests/test_models.py

```diff
@@ -9,13 +9,13 @@ class TestTeamModel(TestCase):
     def setUp(self):
         self.battle = mommy.make("battles.Battle")
         self.trainer = mommy.make("users.User")
-        self.pokemon_set = mommy.prepare("pokemon.Pokemon", _quantity=3, make_m2m=True)
+        self.pokemon_1 = mommy.make("pokemon.Pokemon", id=1)
+        self.pokemon_2 = mommy.make("pokemon.Pokemon", id=2)
+        self.pokemon_3 = mommy.make("pokemon.Pokemon", id=3)
+        self.team = (self.pokemon_1.id, self.pokemon_2.id, self.pokemon_3.id)
 
     def test_create_team(self):
         item = Team.objects.create(trainer=self.trainer, battle=self.battle)
-        pkn_id = []
-        for pokemon in self.pokemon_set:
-            pkn_id.append(pokemon.id)
-        item.team.set(pkn_id)
+        item.team.set(self.team)
```

Comment:
I see what you did here. I will not ask you to change this, but the naming could be improved here.

```
team = Team.objects.create(trainer=self.trainer, battle=self.battle)
team.deck.set(self.deck) 
# or 
team.pokemon.set(self.pokemon)
```